configure

```diff
@@ -16,10 +16,11 @@ fi
 ## Parse the args, only looking for -camldir
 ## We avoid using shift to keep "$@" intact
 
+cmd=$ocaml
 last=
 for i; do
    case $last in
-       -camldir|--camldir) cmd="$i/ocaml"; break;;
```

Comment:
But I don't understand the remark either! `configure -help` shows no option with two dashes, except `--help` itself which is by convention. As for `coqtop -help`, only the recent `--print-version`, `--xml_format=Ppcmds`, `--help-XML-protocol` have exclusively one dash while the optional `--version` is somehow also by convention).

Obviously, a choice has to be made between a uniformity policy and a GNU-style policy. (We could actually also adopt a "diversity" policy, but my understanding was that the general feeling was rather going towards uniformity.)

So, do you mean that the three exclusive `--` options should a minima exist also with a single dash, and be exclusively documented as such? (Keeping the two dashes only for compatibility.)

Or do you mean that all long-style options should exist with `--` too? This would indeed be the most flexible and open-minded approach, while still satisfying uniformity.